src/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -182,6 +183,54 @@ public static int split(String aString, String[] tokens, char delim) {
     }
 
     /**
+     * Split the string into tokens given by the delimiter.  Will use a 
+     * Vector<String> to allow for dynamically resizing of the tokens array.
+     * Note that with careful pre-allocation of the tokens Vector, this should
+     * be no less efficient than the split(String, String[] char) function.
+     * Also, with this function, we do not need a split(String, Vector<String>, char, bool)
+     * function, as there is no maximum number of tokens.
+     * 
+     * @param sSrting the string to split
+     * @param tokens  a Vector<String> to hold the parsed tokens
+     * @param delim   character that delimits tokens
+     * @return the number of tkens parsed
+     */
+    public static int split(String aString, Vector<String> tokens, char delim) {
+        
+    	int nTokens = 0;
+        int start = 0;
+        int end = aString.indexOf(delim);
+        
+        // Empty out our answer, but preserve the memory allocation
+        tokens.clear();
+        
+        while(end >= 0){
+        	// Only add non-empty tokens
+        	if(start != end){
+        		tokens.add(aString.substring(start, end));
```

Comment:
For Vector, add appends to the end of the vector. So, for this case, the first add will create element 10001;